ladybug_comfort/collection/solarcal.py

```diff
@@ -23,6 +23,26 @@
 class _SolarCalBase(ComfortCollection):
     """Base class used by all objects that use SolarCal with Data Collections."""
 
+    def __init__(self, location, fraction_body_exposed=None, floor_reflectance=None,
```

Comment:
Do we want to have `__slots__` for this class?

Reply:
All of ladybug_comfort was written back before I really got into using `__slots__` but it's a good idea, especially given that a lot of data will probably be flowing though these classes in the comfort mapping workflows. I'll add it now.

Reply:
Done.